### PR TITLE
use config in ioctl

### PR DESCRIPTION
**Pull request by Frankonly**
use config in ioctl

### CODE REVIEW

File: cli/ioctl/cmd/account/accountbalance.go [hunk 1]
```diff
@@ -32,13 +32,17 @@ var accountBalanceCmd = &cobra.Command{
...
-	accountBalanceCmd.Flags().StringVarP(&hostAddress, "host", "s", "127.0.0.1:8080", "host address of node")
 	AccountCmd.AddCommand(accountBalanceCmd)
 }
 
 // balance gets balance of an IoTex Blockchian address
 func balance(args []string) string {
-	conn, err := grpc.Dial(hostAddress, grpc.WithInsecure())
+	endpoint := config.GetEndpoint()
+	if endpoint == "" {
+		log.L().Error("get empty endpoint")
```
Comment (raullenchai) [inline cli/ioctl/cmd/account/accountbalance.go]: `use "ioctl config" to config endpoint first.` -- more user friendly

File: cli/ioctl/cmd/action/actionhash.go [hunk 2]
```diff
@@ -34,13 +31,17 @@ var actionHashCmd = &cobra.Command{
...
 	ActionCmd.AddCommand(actionHashCmd)
 }
 
 // getActionByHash gets balance of an IoTex Blockchain address
 func getActionByHash(args []string) string {
-	conn, err := grpc.Dial(hostAddress, grpc.WithInsecure())
+	endpoint := config.GetEndpoint()
+	if endpoint == "" {
+		log.L().Error("get empty endpoint")
+		return "get empty endpoint"
```
Comment (raullenchai) [inline cli/ioctl/cmd/action/actionhash.go]: same here

File: Makefile [hunk 3]
```diff
@@ -166,3 +166,10 @@ nightlybuild:
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_MINICLUSTER) -v ./tools/minicluster
 	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib:$(PWD)/crypto/lib/blslib
 	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
+
+.PHONY: testcli
```
Comment (raullenchai) [inline Makefile]: This is added to test ioctl using a local bc server?

Comment (Frankonly) [reply Makefile]: Yes. It runs a bc server with different config.

File: Makefile [hunk 4]
```diff
@@ -158,4 +158,4 @@ nightlybuild:
 	$(ECHO_V)rm -rf *trie*.db
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_MINICLUSTER) -v ./tools/minicluster
 	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib:$(PWD)/crypto/lib/blslib
-	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
+	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
```
Comment (raullenchai) [inline Makefile]: remove this whitespace change

**Pull request merged**